### DISCUSSION TITLE
about HoldBrowser?

### DISCUSSION THREAD

**Discussion by langzime** [fltk-rs/fltk-rs#1118] [Q&A] [answered]
i want write a chat list use ”HoldBrowser “，  How do i hide some information like “userid” in HoldBrowser’s  item?

**Comment by MoAlyousef** [comment, marked as answer]
You would change the width of the corresponding column using HoldBrowser::set_column_widths(). 
It appears  FLTK uses 0 as a sentinel value to mark the end of the passed array, so you can try setting the width to -1.
